trollmoves/tests/test_movers.py

```diff
@@ -88,6 +89,30 @@ def test_s3_copy_file_to_base(S3FileSystem):
     S3FileSystem.return_value.put.assert_called_once_with(ORIGIN, "data-bucket/filename.ext")
 
 
+@patch('trollmoves.movers.S3FileSystem')
+def test_s3_get_destination_from_string(S3FileSystem):
```

Comment:
This should be 3 separate test cases.

The name is also too generic, also other tests passing strings to the mover. Maybe something like `test_s3mover_destination_FOO()` where `FOO` describes which input or feature is being tested.